benchmarks/kane3.py

```diff
@@ -54,7 +79,17 @@
 print "Subs"
 cg = cg.subs(dict(zip(consts, range(1, len(consts)+1))))
 cg = cg.subs(dict(zip(syms, range(1, len(syms)+1))))
-print "Eval derivative"
+print "SymPy/SymEngine Eval derivative"
 print cg.n(real=True)
-print cg.n(200)
+print cg.n(200, real=True)
```

Comment:
We should raise an exception instead of returning NaN. The NaN are very hard to debug, since you don't know where they are coming from.